dataactcore/scripts/pipeline/get_fsrs_updates.py

```diff
@@ -69,12 +69,24 @@ def get_award_updates(mod_date):
                 WHERE ut.fain = pf.fain
             )
             GROUP BY fain),
-        only_base AS (SELECT pf.*, base_date, earliest_start, latest_end, currently_active, obligation_sum
-            FROM published_fabs AS pf
-            JOIN grouped_values AS gv
-                ON gv.fain = pf.fain
-                AND gv.max_updated = pf.updated_at
-                AND pf.record_type != 1)
+        only_base AS (
+            SELECT *
+            FROM (SELECT pf.*,
+                    base_date,
+                    earliest_start,
+                    latest_end,
+                    currently_active,
+                    obligation_sum,
+                    ROW_NUMBER() OVER (PARTITION BY
+                        UPPER(pf.fain)
+                        ORDER BY updated_at DESC
+                    ) AS row_num
+                FROM published_fabs AS pf
+                JOIN grouped_values AS gv
+                    ON gv.fain = pf.fain
+                    AND gv.max_updated = pf.updated_at
+                    AND pf.record_type != 1) duplicates
```

Comment:
Took me a minute to visually see where duplicates was coming from

```suggestion
                    AND pf.record_type != 1) AS duplicates
```